{
  "uid" : "61ab0f55d5eeb68f",
  "name" : "test_login_failure",
  "fullName" : "test_feature_story.TestLogin#test_login_failure",
  "historyId" : "a58c5550e5c40eb80abda2e84e87077a",
  "time" : {
    "start" : 1616501477475,
    "stop" : 1616501477479,
    "duration" : 4
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert '1' == 1",
  "statusTrace" : "self = <test_feature_story.TestLogin object at 0x0000000003752580>\n\n    @allure.story(\"密码缺失\")\n    def test_login_failure(self):\n        # with allure.step  标记测试步骤\n        with allure.step(\"点击用户名\"):\n            print(\"输入用户名\")\n        with allure.step(\"点击密码\"):\n            print(\"输入密码\")\n        print(\"点击登录\")\n        with allure.step(\"点击登录之后登录失败\"):\n>           assert '1' == 1\nE           AssertionError: assert '1' == 1\n\ntest_feature_story.py:31: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert '1' == 1",
    "statusTrace" : "self = <test_feature_story.TestLogin object at 0x0000000003752580>\n\n    @allure.story(\"密码缺失\")\n    def test_login_failure(self):\n        # with allure.step  标记测试步骤\n        with allure.step(\"点击用户名\"):\n            print(\"输入用户名\")\n        with allure.step(\"点击密码\"):\n            print(\"输入密码\")\n        print(\"点击登录\")\n        with allure.step(\"点击登录之后登录失败\"):\n>           assert '1' == 1\nE           AssertionError: assert '1' == 1\n\ntest_feature_story.py:31: AssertionError",
    "steps" : [ {
      "name" : "点击用户名",
      "time" : {
        "start" : 1616501477475,
        "stop" : 1616501477476,
        "duration" : 1
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : false
    }, {
      "name" : "点击密码",
      "time" : {
        "start" : 1616501477476,
        "stop" : 1616501477476,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : false
    }, {
      "name" : "点击登录之后登录失败",
      "time" : {
        "start" : 1616501477477,
        "stop" : 1616501477477,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: assert '1' == 1\n",
      "statusTrace" : "  File \"D:\\PycharmProjects\\hogwarts\\test_feature_story.py\", line 31, in test_login_failure\n    assert '1' == 1\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "stepsCount" : 0,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "210c545f73bc70e6",
      "name" : "stdout",
      "source" : "210c545f73bc70e6.txt",
      "type" : "text/plain",
      "size" : 42
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "stepsCount" : 3,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "登录模块"
  }, {
    "name" : "story",
    "value" : "密码缺失"
  }, {
    "name" : "suite",
    "value" : "test_feature_story"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "PC-201705071250"
  }, {
    "name" : "thread",
    "value" : "8860-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_feature_story"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "61ab0f55d5eeb68f.json",
  "parameterValues" : [ ]
}